src/Chunk.h

```diff
@@ -392,18 +392,37 @@ class cChunk :
 	{
 		return m_ChunkData.GetMeta(a_RelX, a_RelY, a_RelZ);
 	}
-	inline void       SetMeta(int a_RelX, int a_RelY, int a_RelZ, NIBBLETYPE a_Meta)
+	inline void  SetMeta(int a_RelX, int a_RelY, int a_RelZ, NIBBLETYPE a_Meta)
```

Comment:
I'll do it your way. Readability is preferred, especially if it turns out to be a micro optimization or if compilers optimize this away. If someone comes up with benchmarks in favor of a dedicated function, we'll later just use function overloading instead of a different brand new function. This keeps the Lua Api clean and saves an export of an additional function.